R/as_sass.R

```diff
@@ -105,6 +105,11 @@ as_sass_.list <- function(input) {
   paste0("$", input_names, ": ", input_values, ";", collapse = "\n")
 }
 
+as_sass_.sass_layer <- function(input) {
+  sass <- as_sass_(list(input$pre, input$post))
+  attachDependencies(as_html(sass, "sass"), input$deps)
```

Comment:
Dependencies don't need to be attached to the result, `sass::sass` is the consumer of these objects and it doesn't care

Reply:
Should `sass::sass()` attempt to retrieve and process the dependencies? It feels a bit odd to me that we'd have `sass_layers()` understand and accumulate dependencies, but then ultimately do nothing useful with that knowledge in `sass::sass()`